pilot/pkg/config/kube/gateway/testdata/invalid.status.yaml.golden

```diff
@@ -433,3 +433,28 @@ status:
       name: gateway
       namespace: istio-system
 ---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  creationTimestamp: null
+  name: no-backend
+  namespace: default
+spec: null
+status:
+  parents:
+  - conditions:
+    - lastTransitionTime: fake
+      message: Route was valid
```

Comment:
should we be reporting a status here?

Reply:
I'm not sure.. I would keep it as before until the spec clearly says this is invalid